web/admin/src/Admin.js

```diff
@@ -69,34 +72,37 @@ export default class Admin extends PureComponent {
+              {sessions[sessionId].isDisplayable ? null : <p>Please Rename Session</p>}
               <button className="secondary" onClick={this.deleteSession}>Delete Session</button>
             </label>
             <div className="session">
               <Questions
                 questions={this.questionsForCurrentSession()}
                 questionsRef={this.questionsRef()}
+                resetHelpText={this.resetHelpText}
+                currentIndex={this.state.currentIndex}
                 renderFooter={() => (
                   <footer>
-                    <div><label>Time per question: <input type="number" value={sessions[sessionId].secondsPerQuestion} onChange={this.onSecondsChange} /> seconds</label></div>
+                    <div><label>Time per question: <input type="number" max="60" value={sessions[sessionId].secondsPerQuestion} onChange={this.onSecondsChange} /> seconds</label></div>
                     <div>
-                      <button onClick={() => this.addQuestion(this.state.sessionId)}>Add New Question</button>
+                      <button onClick={() => this.addQuestion(this.state.sessionId)}>{this.state.helpText}</button>
                     </div>
                   </footer>
                 )}
               />
-              <div className="presentation-container">
-                <div className="presentation-side">
-                  <iframe className="big-screen-container" src={this.bigScreenUrl()} title="presentation" />
-                  <div className="presentation-overlays">
-                    <div>Presentation Screen <button className="overlay-button" onClick={this.launchPresentation} disabled={launchDisabled || !this.bigScreenUrl()}>Launch in new tab</button></div>
-                  </div>
+            </div>
+            <div className="presentation-container">
+              <div className="presentation-side">
+                <iframe className="big-screen-container" src={this.bigScreenUrl()} title="presentation" />
+                <div className="presentation-overlays">
+                  <div>Presentation Screen <button className="overlay-button" onClick={this.launchPresentation} disabled={launchDisabled || !this.bigScreenUrl()}>Launch Presentation</button></div>
                 </div>
-                <div className="presentation-side">
-                  <PresentationDriver fbc={this.props.fbc} session={sessions[sessionId]} questions={this.questionsForCurrentSession()} users={users} />
-                  <div className="presentation-overlays">
-                    <div>Up Next</div>
-                  </div>
+              </div>
+              <div className="presentation-side">
+                <PresentationDriver fbc={this.props.fbc} session={sessions[sessionId]} saveCurrentIndex={this.saveCurrentIndex} questions={this.questionsForCurrentSession()} users={users} />
```

Comment:
It looks like the index of the current question from the presentation (actually running a trivia game) is being sent out to the list of questions that can be edited.  Is this to prevent the CMS user from deleting a question that is currently showing in a trivia game?  If so, this will only prevent one case (you could delete it from another window, etc), and there are other weird things that can happen when deleting other questions for a game already in progress.  IMO I would not try to block such actions and complicate things.

Reply:
Would you suggest we prevent users from making any deletes during an active game?

Reply:
No... someone might decide that an upcoming question should not be shown at the last minute and delete it before we get there.

Reply:
Maybe we can disallow deleting the current question or any previous questions?

Reply:
To enable the fix in the above comment (disabling the button) Is there a better way to get the index of the current question in the game?

Reply:
I think this turns out to be a fine way to bubble the index of the current question up.  It won't do anything about other tabs being open, but it will make it harder for the presenter to 👣 🔫